### ISSUE TITLE
Progress of step "recognizing text" no longer gets logged

### ISSUE THREAD

**Issue by Eggerd** [naptha/tesseract.js#655]
**Describe the bug**
Progress of step `recognizing text` doesn't get logged anymore since v3.0.0 (seems to be the case since 13b95f6371cf6867e1285832aca6aff6dff84244).

**To Reproduce**
1. Checkout v2.1.5, install dependencies, and start the dev server
2. Open the localhost browser demo in any browser
3. Click the *sample image* button and wait until the recognized text is being shown
4. Note that progress of step `recognizing text` is being reported
5. Repeat all steps above with v3.0.0 and note that progress of `recognizing text` no longer gets reported

**Expected behavior**
Progress of `recognizing text` should get reported by the logger after step `initialized api` completed.

**Screenshots**
| v2.1.5 | v3.0.0 - v3.0.2 |
| --- | --- |
| ![v2-1-5](https://user-images.githubusercontent.com/904567/187534039-f9c3e161-34c2-4557-9341-9605733eff9c.png) | ![v3-0-0](https://user-images.githubusercontent.com/904567/187534045-82b30f41-495a-4486-a333-2f4c0d2ce12d.png) |

**Desktop**
 - OS: Windows 10
 - Browser: Chrome 104.0.5112.102 / Firefox 104.0.1
 - Version: >3.0.0

**Comment by Balearica** [comment]
Thanks for reporting this, as well as providing enough detail to replicate.  I confirmed that this was caused by the new engine build in [Tesseract.js-core](https://github.com/naptha/tesseract.js-core), as when using Tesseract.js-core directly (cutting out all the code in this repo) the results are the same as you describe above.  Currently not sure what could have caused this, but will investigate in the future to the extent I have time.
